movie-reviews/src/app/components/core/RadioGroupComponent/radio-styles.css

```diff
@@ -0,0 +1,10 @@
+.chakra-radio__control {
```

Comment:
Maybe not a part of this homework, but the Chakra way to use a radio button that does not look like a radio button would be: https://v2.chakra-ui.com/docs/components/radio#custom-radio-buttons

Reply:
Yeah, thought of implementing that but I'm way behind on schedule already 😬